app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/DeviceCommandsTest.java

```diff
@@ -824,4 +825,11 @@ public void shouldExtractDeviceInformation() throws JSONException {
         assertEquals(info.getString("platformVersion"), Build.VERSION.RELEASE);
         assertTrue(info.getJSONArray("networks").length() > 0);
     }
+
+    @Test
+    public void shouldExtractDeviceApplications() throws JSONException {
```

Comment:
the terminology used is a bit confusing to me
ExtractDeviceApplications vs getListApplications vs GetPackages
is it possible to follow the same terminology everywhere?

Reply:
Done the formatting